hydro_lang/src/builder/deploy.rs

```diff
@@ -183,7 +183,11 @@ impl<'a, D: Deploy<'a>> DeployFlow<'a, D> {
 
     fn extra_stmts(&self, env: &<D as Deploy<'a>>::CompileEnv) -> BTreeMap<usize, Vec<syn::Stmt>> {
         let mut extra_stmts: BTreeMap<usize, Vec<syn::Stmt>> = BTreeMap::new();
-        for &c_id in self.clusters.keys() {
+
+        let mut all_clusters_sorted = self.clusters.keys().collect::<Vec<_>>();
+        all_clusters_sorted.sort();
+
+        for &c_id in all_clusters_sorted {
```

Comment:
Would it be possible to make `self.clusters` a `BTreeMap` instead of `HashMap`?

Reply:
Yes although we don't actually care about the order anywhere else (even when we iterate over the elements), so was thinking it's cleaner to localize it?